reactor-netty-core/src/main/java/reactor/netty/transport/NameResolverProvider.java

```diff
@@ -0,0 +1,591 @@
+		 * @return {@code this}
+		 */
+		NameResolverSpec disableOptionalRecord(boolean disable);
+
+		/**
+		 * Specifies whether this resolver has to send a DNS query with the recursion desired (RD) flag set.
+		 * By default this is enabled.
+		 *
+		 * @param disable true if RD flag is not set
+		 * @return {@code this}
+		 */
+		NameResolverSpec disableRecursionDesired(boolean disable);
+
+		/**
+		 * Sets the capacity of the datagram packet buffer (in bytes).
+		 * Default to {@link Build#DEFAULT_MAX_PAYLOAD_SIZE}.
+		 *
+		 * @param maxPayloadSize the capacity of the datagram packet buffer
+		 * @return {@code this}
+		 * @throws IllegalArgumentException if {@code maxPayloadSize} is not positive
+		 */
+		NameResolverSpec maxPayloadSize(int maxPayloadSize);
+
+		/**
+		 * Sets the maximum allowed number of DNS queries to send when resolving a host name.
+		 * Default to {@link Build#DEFAULT_MAX_QUERIES_PER_RESOLVE}.
+		 *
+		 * @param maxQueriesPerResolve the max number of queries
+		 * @return {@code this}
+		 * @throws IllegalArgumentException if {@code maxQueriesPerResolve} is not positive
+		 */
+		NameResolverSpec maxQueriesPerResolve(int maxQueriesPerResolve);
+
+		/**
+		 * Sets the number of dots which must appear in a name before an initial absolute query is made.
+		 * Default to {@link Build#DEFAULT_NDOTS} which determines the value from the OS on Unix
+		 * or uses the value {@code 1}.
+		 *
+		 * @param ndots the ndots value
+		 * @return {@code this}
+		 * @throws IllegalArgumentException if {@code ndots} is less than -1
+		 */
+		NameResolverSpec ndots(int ndots);
+
+		/**
+		 * Sets the timeout of each DNS query performed by this resolver (resolution: milliseconds).
+		 * Default to {@link Build#DEFAULT_QUERY_TIMEOUT}.
+		 *
+		 * @param queryTimeout the query timeout (resolution: milliseconds)
+		 * @return {@code this}
+		 */
+		NameResolverSpec queryTimeout(Duration queryTimeout);
+
+		/**
+		 * Sets the list of the protocol families of the address resolved.
+		 *
+		 * @param resolvedAddressTypes the address types
+		 * @return {@code this}
+		 */
+		NameResolverSpec resolvedAddressTypes(ResolvedAddressTypes resolvedAddressTypes);
+
+		/**
+		 * Enables an {@link AddressResolverGroup} of {@link DnsNameResolver}s that supports random selection
+		 * of destination addresses if multiple are provided by the nameserver.
+		 * See {@link RoundRobinDnsAddressResolverGroup}.
+		 * Default to {@link DnsAddressResolverGroup}.
+		 *
+		 * @return {@code this}
+		 */
+		NameResolverSpec roundRobinSelection(boolean enable);
+
+		/**
+		 * Performs the communication with the DNS serversSets on the given {@link EventLoopGroup}.
+		 *
+		 * @param eventLoopGroup the {@link EventLoopGroup}
+		 * @return {@code this}
+		 */
+		NameResolverSpec runOn(EventLoopGroup eventLoopGroup);
+
+		/**
+		 * Performs the communication with the DNS serversSets on a supplied {@link EventLoopGroup}
```

Comment:
typo `serverSets` instead of `server` ? (also found in other javadocs below)

Reply:
fixed